UpsertSpecialServiceActivationActivity/__tests__/handler.test.ts

```diff
@@ -50,17 +41,23 @@ describe("UpsertSpecialServiceActivationActivity", () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
-  it("should return failure if an error occurs during input decode retrieve", async () => {
+  it("should throw if an error occurs during input decode retrieve", async () => {
     const upsertSpecialServiceActivationActivity = getUpsertSpecialServiceActivationActivityHandler(
       servicesClientMock
     );
-    const response = await upsertSpecialServiceActivationActivity(context, {
-      aaa: "wrongActivityInput"
-    });
-    expect(response.kind).toBe("FAILURE");
+    await pipe(
+      TE.tryCatch(
+        () =>
+          upsertSpecialServiceActivationActivity(context, {
+            aaa: "wrongActivityInput"
+          }),
+        E.toError
+      ),
+      TE.mapLeft(ex => expect(ex).toBeDefined())
```

Comment:
Always test both the side of the TE. For example : 
```suggestion
      TE.bimap(
        ex => expect(ex).toBeDefined(),
        _ => fail("Unexpected value")
      )
```
But something is wrong in here. A wrong input decode should return a permanent failure not a transient one with related throw of the Activity 🤔